common/types/data_connections.ts

```diff
@@ -35,11 +35,12 @@ export interface Acceleration {
 }
 
 export interface AssociatedObject {
+  tableName: string;
```

Comment:
@RyanL1997 @sejli are you fine with this additional field in associated object? using it for defining the table name in skipping index redirection

Reply:
Sounds good to me, as long as it's updated wherever this type is used.